api/net/Areas/Subscriber/Controllers/ContentController.cs

```diff
@@ -72,7 +72,7 @@ public class ContentController : ControllerBase
     [Produces(MediaTypeNames.Application.Json)]
     [ProducesResponseType(typeof(IPaged<Services.Models.Content.ContentModel>), (int)HttpStatusCode.OK)]
     [SwaggerOperation(Tags = new[] { "Content" })]
-    public async Task<IActionResult> FindWithElasticsearchAsync()
+    public async Task<IActionResult> FindWithElasticsearchAsync([FromQuery] string? index)
```

Comment:
Don't implement this way.  Instead add a boolean field to the ContentFilter.  This way we don't have to manage index names on the client app and have to deal with invalid index names.  I'll probably change the editor implementation at some point too.